libkbfs/interfaces.go

```diff
@@ -738,7 +738,7 @@ type MDOps interface {
 	// which may not yet be reflected in the MD if the TLF hasn't been rekeyed since it
 	// entered into a conflicting state.
 	GetLatestHandleForTLF(ctx context.Context, id TlfID) (
-		*BareTlfHandle, error)
+		BareTlfHandle, error)
```

Comment:
I think it was just to be consistent with the other interface functions that take `*TlfHandle`.  Probably no real reason.